src/block_cache/block_chunkcache.c

```diff
@@ -587,11 +595,12 @@ void
 __wt_chunkcache_remove(
   WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid, wt_off_t offset, uint32_t size)
 {
+    bool valid;
+    size_t already_removed, remains_to_remove, removable_in_chunk, size_removed;
+    uint64_t bucket_id;
```

Comment:
Nit: As per the WT convention, shouldn't primitive variables be declared at the end?

Reply:
Oops, I was a bit too vigorous in where I asked Emacs to sort lines - thanks. Fixed in fee8306